File: aula#7/AbstractFactory_java/Pizza/PizzaBolonha.java
package Pizza;

//ConcreteProductB2
class PizzaBolonha implements Pizza {

 @Override
 public void servir(PizzaVeg pizzaVeg) {
     System.out.println(getClass().getSimpleName() + " é servida com bolonha na " + pizzaVeg.getClass().getSimpleName());
 }
}

File: aula#7/pizza/PizzaBolonha.py
from abc import ABCMeta, abstractmethod
from Pizza import Pizza

# ProdutoConcreto
class PizzaBolonha(Pizza):

    def servir(self, PizzaVeg):
        print(f'{type(self).__name__} é servida com bolonha na {type(PizzaVeg).__name__}')

